api/tests/test_cloud_record_summary.py

```diff
@@ -36,10 +37,14 @@ def test_cloud_record_summary_get_IAM_fail(self):
                                            "Day",
                                            "Month",
                                            "Year"]):
+
             test_client = Client()
-            response = test_client.get('/api/v1/cloud/record/summary?'
-                                       'group=TestGroup&'
-                                       'from=20000101&to=20191231',
+            get_url = ''.join((reverse('CloudRecordSummaryView'),
```

Comment:
Style point - you shouldn't use an action, like "get", in a variable name as calling that variable doesn't perform that action. Should probably just be `url`.

However, separating it out to a separate variable seems a bit unnecessary as line 113 and 178, for example, seem to be reasonably clear. Either split them all out to separate variables (and change the name) or fold them all in. Up to you, but should be consistent.